src/HandlerResolver.php

```diff
@@ -75,6 +76,8 @@ private function laravelApp(): Application
                 is_object($this->laravelApp) ? get_class($this->laravelApp) : gettype($this->laravelApp),
             ));
         }
+        $kernel = $this->laravelApp->make(Kernel::class);
+        $kernel->bootstrap();
```

Comment:
Forgotten statements in #43: we can't properly retrieve services from the container if Laravel is not bootstrapped. This is something that was done previously in `runtime.php` (and in the existing Laravel bridge), I just forgot about it 👼